packages/amplify-provider-awscloudformation/src/push-resources.ts

```diff
@@ -904,7 +907,7 @@ async function formNestedStack(
   let authResourceName: string;
 
   const { APIGatewayAuthURL, NetworkStackS3Url, AuthTriggerTemplateURL } = amplifyMeta.providers[constants.ProviderName];
-
+  const { envName } = stateManager.getLocalEnvInfo();
```

Comment:
I think this is used during init as well when the localEnvInfo file isn't present. Can you please make sure init and checkout env doesn't fail with this change? 'context.exeInfo.localEnvInfo.envName' always has the info.

Reply:
Why are we pushing at init and checkout?

Reply:
I think if you do a force push on an init with already existing amplify project - that's when it does init + push. Wouldn't recommend this refactor at this point.